### PR TITLE
[GITHUB-3] Add Support for Version 0.9.x

### PR DESCRIPTION
**Pull request by wojtek-oledzki**
Added support for (and only for) version >= 0.9

### CODE REVIEW

File: defaults/main.yml [hunk 1]
```diff
@@ -42,6 +50,7 @@ kong:
     - ssl
     - tcp-log
     - udp-log
-  proxy_listen: "0.0.0.0:8000"
-  proxy_listen_ssl: "0.0.0.0:8443"
-  version: 0.8.3
+
+  systemd:
```
Comment (wojtek-oledzki) [inline defaults/main.yml]: I need to remove that :) (spoiler alert)

File: .travis.yml [hunk 2]
```diff
@@ -11,16 +11,18 @@ language: python
 python: "2.7"
 
 env:
-  - ANSIBLE_INSTALL_VERSION=1.9.4
-  - ANSIBLE_INSTALL_VERSION=2.0.0.2
   - ANSIBLE_INSTALL_VERSION=2.0.2.0
-  
+  - ANSIBLE_INSTALL_VERSION=2.1.1.0
```
Comment (lobsterdore) [inline .travis.yml]: Should we use the latest 2.1 version here? 2.1.4.0.

File: .version [hunk 3]
```diff
@@ -0,0 +1 @@
+v1.0
```
Comment (lobsterdore) [inline .version]: Shall we go for v1.1? Just in case someone is using the role.

File: tasks/install.yml [hunk 4]
```diff
@@ -1,30 +1,19 @@
...
-  with_items: "{{ kong.dependencies }}"
+  apt_repository:
+    repo: "deb https://dl.bintray.com/mashape/kong-ubuntu-{{ ansible_distribution_release }}-{{ kong.version | regex_replace('(^[0-9]+\\.[0-9]+).*$', '\\1.x') }}
+      {{ ansible_distribution_release }} main"
+    state: present
 
-# This seems to fail often, retry a few times
-- name: Download kong
+  # We still do support Ansible 1.9 hence "command" with --allow-unauthenticated
+  # instead of apt module :()
```
Comment (lobsterdore) [inline tasks/install.yml]: Do we still support Ansible 1.9?

File: tests/vagrant/Vagrantfile [hunk 5]
```diff
@@ -29,7 +29,14 @@ Vagrant.configure(VAGRANTFILE_API_VERSION) do |config|
 	# Test "configure" tag
 	config.vm.provision "ansible" do |ansible|
 		ansible.playbook = "../test.yml"
-		ansible.tags = [ "configure", "assert" ]
+		ansible.tags = [ "configure" ]
+		ansible.verbose = "#{ENV['VERBOSE']}" if ENV['VERBOSE']
+	end
+
+	# Test "configure" tag
```
Comment (lobsterdore) [inline tests/vagrant/Vagrantfile]: Should be "assert"

File: .travis.yml (hunk 2 above)
Comment (wojtek-oledzki) [reply .travis.yml]: yep

File: .version (hunk 3 above)
Comment (wojtek-oledzki) [reply .version]: yep :)

File: tasks/install.yml (hunk 4 above)
Comment (wojtek-oledzki) [reply tasks/install.yml]: I will update the comment. The `allow_unauthenticated` was added in 2.1 so we still need it :(

File: tests/vagrant/Vagrantfile (hunk 5 above)
Comment (wojtek-oledzki) [reply tests/vagrant/Vagrantfile]: :) fixing

File: tasks/install.yml [hunk 6]
```diff
@@ -1,30 +1,19 @@
...
-  until: kong_download_data | success
-  retries: 5
-  delay: 10
-
-- name: Install kong
-  become: yes
-  apt:
-    deb: "/var/tmp/kong-{{ kong.version }}.deb"
-    state: installed
+  command: "apt-get -y --allow-unauthenticated install 'kong={{ kong.version }}'"
```
Comment (lobsterdore) [inline tasks/install.yml]: Can you try using the force option of the apt module instead of this?

File: tasks/install.yml [hunk 7]
```diff
@@ -46,3 +35,4 @@
   service:
     name: kong
     enabled: yes
+    state: stopped
```
Comment (lobsterdore) [inline tasks/install.yml:37]: :+1:

File: tasks/install.yml (hunk 6 above)
Comment (wojtek-oledzki) [reply tasks/install.yml]: why `--force-yes`? We only want to acknowledge absence of GPG signature

Comment (wojtek-oledzki) [reply tasks/install.yml]: oh, `force` on `apt` module ... checking if it's in 2.0...

Comment (wojtek-oledzki) [reply tasks/install.yml]: it's even in 1.9 :), changing now...

**Review by lobsterdore — APPROVED** [review]
LGTM :+1:

**Pull request merged**